presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -350,6 +346,12 @@ public PlanMatchPattern withAlias(Optional<String> alias, RvalueMatcher matcher)
         return this;
     }
 
+    public PlanMatchPattern withNumberOfOutputColumns(int numberOfSymbols)
```

Comment:
Avoid moving the method at the same time you make changes to the logic inside it. It makes it harder to see what's different.